sdk/apimanagement/Azure.ResourceManager.ApiManagement/src/Customized/ArmApiManagementModelFactory.cs

```diff
@@ -0,0 +1,405 @@
+        /// <param name="license"> License information for the API. </param>
+        /// <param name="sourceApiId"> API identifier of the source API. </param>
+        /// <param name="displayName"> API name. Must be 1 to 300 characters long. </param>
+        /// <param name="serviceUri"> Absolute URL of the backend service implementing this API. Cannot be more than 2000 characters long. </param>
+        /// <param name="path"> Relative URL uniquely identifying this API and all of its resource paths within the API Management service instance. It is appended to the API endpoint base URL specified during the service instance creation to form a public URL for this API. </param>
+        /// <param name="protocols"> Describes on which protocols the operations in this API can be invoked. </param>
+        /// <param name="apiVersionSet"> Version set details. </param>
+        /// <returns> A new <see cref="ApiManagement.ApiData"/> instance for mocking. </returns>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static ApiData ApiData(ResourceIdentifier id = null, string name = null, ResourceType resourceType = default, SystemData systemData = null, string description = null, AuthenticationSettingsContract authenticationSettings = null, SubscriptionKeyParameterNamesContract subscriptionKeyParameterNames = null, ApiType? apiType = null, string apiRevision = null, string apiVersion = null, bool? isCurrent = null, bool? isOnline = null, string apiRevisionDescription = null, string apiVersionDescription = null, ResourceIdentifier apiVersionSetId = null, bool? isSubscriptionRequired = null, Uri termsOfServiceUri = null, ApiContactInformation contact = null, ApiLicenseInformation license = null, ResourceIdentifier sourceApiId = null, string displayName = null, Uri serviceUri = null, string path = null, IEnumerable<ApiOperationInvokableProtocol> protocols = null, ApiVersionSetContractDetails apiVersionSet = null)
+        {
+            protocols ??= new List<ApiOperationInvokableProtocol>();
+
+            return new ApiData(
+                id,
+                name,
+                resourceType,
+                systemData,
+                description,
+                authenticationSettings,
+                subscriptionKeyParameterNames,
+                apiType,
+                apiRevision,
+                apiVersion,
+                isCurrent,
+                isOnline,
+                apiRevisionDescription,
+                apiVersionDescription,
+                apiVersionSetId,
+                isSubscriptionRequired,
+                termsOfServiceUri.AbsoluteUri.ToString(),
+                contact,
+                license,
+                sourceApiId,
+                displayName,
+                serviceUri.AbsoluteUri.ToString(),
+                path,
+                protocols?.ToList(),
+                apiVersionSet,
+                serializedAdditionalRawData: null);
```

Comment:
we should call the generated version of method with the same name in this model factory such as:
```suggestion
            return ApiData(
                id: id,
                name, // please add parameter name for all the parameters to avoid future updates to break this invocation
                resourceType,
                systemData,
                description,
                authenticationSettings,
                subscriptionKeyParameterNames,
                apiType,
                apiRevision,
                apiVersion,
                isCurrent,
                isOnline,
                apiRevisionDescription,
                apiVersionDescription,
                apiVersionSetId,
                isSubscriptionRequired,
                termsOfServiceUri.AbsoluteUri.ToString(),
                contact,
                license,
                sourceApiId,
                displayName,
                serviceUri.AbsoluteUri.ToString(),
                path,
                protocols?.ToList(),
                apiVersionSet);
```

Reply:
same comment should apply to other methods in this customized class as well